config/gni/devtools_image_files.gni

```diff
@@ -11,6 +11,7 @@ devtools_image_files = [
   "chromeMiddle.avif",
   "chromeRight.avif",
   "cssoverview_icons_2x.avif",
+  "favicon.ico",
```

Comment:
AFAICT the images listed here are (1) meant to be referenced by JS, (2) passed through an image optimisation pipeline that I'm not sure is compatible with `.ico` files. We would probably want to define a separate build action somewhere that just copies `favicon.ico` to somewhere under `front_end/`.